client/components/Account.js

```diff
@@ -14,8 +14,12 @@ const Account = ({ user, _destroySession }) => {
       <p>Email: {email} </p>
       <p>Admin: {isAdmin ? 'Yep' : 'Nope'} </p>
       <Link to="/" onClick={()=>{
+          //give a cookie an expiration date in the past in order to delete it
           document.cookie = 'sessionId=; Path=/; Expires=Thu, 01 Jan 1970 00:00:01 GMT;';
-          _destroySession(sessionId)
+          //get rid of the session so you're not automatically logged back in
+          _destroySession(sessionId);
+          //empty your cart if you log out
```

Comment:
Yeah, I meant for this to empty the front-end, redux cart, so that you don't have everything that was in your basket still there after you've logged out (but it is in the DB, associated w your account).
Mostly I was trying to avoid having stuff in your cart when you log out that then gets added _again_ to your cart when you log back in. I figure if the cart is empty after logging out, it's clear enough that that stuff was your "account cart" only.